_topic_map.yml

```diff
@@ -2858,6 +2858,8 @@ Topics:
       File: virt-configuring-sriov-device-for-vms
     - Name: Configuring certificate rotation
       File: virt-configuring-certificate-rotation
+    - Name: Connecting virtual machines to a service mesh
+      File: virt-connecting-vm-to-service-mesh
```

Comment:
@sjhala-ccs 
I was wondering -- did you intend to put the service mesh assembly among the SR-IOV assemblies?